lib/Installation/Partitioner/LibstorageNG/EncryptPasswordPage.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: The class introduces all accessing methods for Encryption Password.
+# Page of Expert Partitioner Wizard, that are common for all the versions of the
+# page (e.g. for both Libstorage and Libstorage-NG).
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+package Installation::Partitioner::LibstorageNG::EncryptPasswordPage;
```

Comment:
nitpicking: I guess the name should be EncryptionPasswordPage, otherwise looks like we encrypt the password, which in this case would be better EncryptDevicePage (which you use [later on](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11276/files#diff-c1a7bee0f0af975a2cc4ec302c9bd6118f54547c1e720fb54578f83d9298ef4aR71)). Don't change it if you don't agree :) , it is how I read it.

Reply:
Funny, that's the first name I gave to the page and then I changed it because the page's header starts with the word "Encrypt". But I guess "Encryption" makes for sense here.